qiskit_experiments/calibration_management/base_calibration_experiment.py

```diff
@@ -149,12 +155,12 @@ def _default_experiment_options(cls):
             result_index (int): The index of the result from which to update the calibrations.
             group (str): The calibration group to which the parameter belongs. This will default
                 to the value "default".
-
+            add_measure_schedules (bool): A boolean that can be set to True (False is the default)
```

Comment:
Can we do this automatically? Namely we can check the attached calibration instance if measure exist. To me such API seems to continuously grow and eventually out of control (like transpile options now).

Reply:
Removed it here: [d5a21be](https://github.com/Qiskit/qiskit-experiments/pull/952/commits/d5a21bed96b3f0d6745eb733b503d6f22940590c) for a follow-up PR.